app/views/godmin/resource/_button_actions.html.erb

```diff
@@ -1,3 +1,3 @@
-<% if policy(@resources).create? %>
+<% if policy(@resource_service.build_resource({})).new? %>
```

Comment:
I did that at first, then I changed it back :) I changed it back because you'd have to remember to default the param if you override the method yourself.